website/source/docs/secrets/pki/index.html.md

```diff
@@ -579,6 +1097,12 @@ If you get stuck at any time, simply run `vault path-help pki` or with a subpath
         `ec` keys. See https://golang.org/pkg/crypto/elliptic/#Curve
         for an overview of allowed bit lengths for `ec`.
       </li>
+      <li>
+        <span class="param">use_csr_common_name</span>
```

Comment:
This seems like it should be defaulted true and always used, I see the logic of this bleeding in elsewhere, but I don't see a use case